lib/model/folderscanner.go

```diff
@@ -25,9 +24,9 @@ type folderScanner struct {
 	delay    chan time.Duration
 }
 
-func newFolderScanner(config config.FolderConfiguration) folderScanner {
+func newFolderScanner(intervalS int) folderScanner {
 	return folderScanner{
-		interval: time.Duration(config.RescanIntervalS) * time.Second,
+		interval: time.Duration(intervalS) * time.Second,
```

Comment:
This looks worse to me, if we end up using some other value later, we'll have to change the signature back.